internal/builder/profiles.go

```diff
@@ -89,6 +89,15 @@ const tmpNotificationPatch = `
     - op: add
       path: /spec/versions/2/schema/openAPIV3Schema/properties/spec/properties/resources/items/properties/kind/enum/-
       value: FluxInstance
+    - op: add
```

Comment:
Don't we want to send any notifications about `ResourceSet` or `ResourceSetInputProvider` as well like we do for `FluxInstance`?

Reply:
Yes but not in this PR. This change is for enabling webhook receivers.